### DISCUSSION TITLE
[spam] edu request for th (Thai) translation

### DISCUSSION THREAD

**Discussion by kkzkk0423** [gsantner/markor#1680] [Ideas]
## Problem && Demand
This is a **feature** request.
I use `th` (Thai) a lot in my daily life. However, the system does not support Thai translation. I hope the system can support Thai. Thanks.

**Comment by YG-yyds** [comment]
This is a good question, and I will continue to pay attention to it.

**Comment by Fantasy-Radix** [comment]
Our team is reviewing the code and will get back to you!

**Comment by kkzkk0423** [comment]
Thanks a lot!

**Comment by gsantner** [comment]
Translation is done here: <https://crowdin.com/project/markor/invite>. So if you want it, translate there.

**Comment by YG-yyds** [comment]
We translated the Thai version using Google Translate, here is the xml file (strings.xml). I hope it will help you. We will pull request later.

```xml
<?xml version="1.0" encoding="utf-8"?>
<resources xmlns:tools="http://schemas.android.com/tools">
</resources>
```

**Comment by gsantner** [comment]
google translate is license wise not acceptable. you need to translate on your own. And do it here, not in xml: <https://crowdin.com/project/markor/invite>

**Comment by YG-yyds** [comment]
Thanks and I will pay attention to it.

**Comment by gsantner** [comment]
see #1679
